__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -1563,64 +1350,53 @@ describe('Omnichannel Chat SDK', () => {
             }
 
-            const optionalParams = {
-                preChatResponse: 'preChatResponse',
-                customContext: {},
-                browser: 'browser',
-                os: 'os',
-                locale: 'locale',
-                device: 'device',
-                initContext: {
-                    preChatResponse: 'override',
-                    customContext: 'override',
-                    browser: 'override',
-                    os: 'override',
-                    locale: 'override',
-                    device: 'override'
-                }
-            }
+            jest.spyOn(chatSDK.OCClient, 'getLWIDetails').mockResolvedValue({
+                State: 'state',
+                ConversationId: 'id',
+                AgentAcceptedOn: 'agentAcceptedOn'
+            });
 
-            jest.spyOn(chatSDK.OCClient, 'getChatToken').mockResolvedValue(Promise.resolve({
-                ChatId: '',
-                Token: '',
-                RegionGtms: '{}'
-            }));
+            await chatSDK.getConversationDetails();
 
-            jest.spyOn(chatSDK.OCClient, 'sessionInit').mockResolvedValue(Promise.resolve());
+            expect(chatSDK.OCClient.getLWIDetails).toHaveBeenCalledTimes(1);
+        });
 
-            await chatSDK.startChat(optionalParams);
+        it('ChatSDK.getConversationDetails() should return "{}" and not throw exception if OCClient.getLWIDetails() fails ', async() => {
+            const chatSDK = new OmnichannelChatSDK(omnichannelConfig);
+            chatSDK.getChatConfig = jest.fn();
 
-            const sessionInitOptionalParams = {
-                initContext: {
-                    preChatResponse: optionalParams.initContext.preChatResponse,
-                    customContext: optionalParams.initContext.customContext,
-                    browser: optionalParams.initContext.browser,
-                    os: optionalParams.initContext.os,
-                    locale: optionalParams.initContext.locale,
-                    device: optionalParams.initContext.device
-                }
+            await chatSDK.initialize();
+
+            jest.spyOn(chatSDK.OCClient, 'getLWIDetails').mockRejectedValue(new Error('Async error message'));
+
+            let errorThrown = false;
+            let conversationDetails;
+            try {
+                conversationDetails = await chatSDK.getConversationDetails();
+            } catch (error) {
+                errorThrown = true;
             }
 
-            expect(chatSDK.OCClient.sessionInit.mock.calls[0][1]).toMatchObject(sessionInitOptionalParams);
+            expect(conversationDetails).toEqual({});
+            expect(chatSDK.OCClient.getLWIDetails).toHaveBeenCalledTimes(1);
+            expect(errorThrown).toBe(false);
         });
 
-        it('[LiveChatV1] ChatSDK.startChat() with authenticatedUserToken should pass it to OCClient.sessionInit() call\'s optional parameters', async() => {
```

Comment:
This test is valid. Its goal is so validate that `authenticatedUserToken` is passed to `sessioninit`. We'd have to remove `v1` dependency on it